models/v1alpha3/relationship/relationship_helper.go

```diff
@@ -50,8 +50,12 @@ func (r *RelationshipDefinition) UpdateStatus(db *database.Handler, status entit
 	return nil
 }
 
-func (r RelationshipDefinition) WriteComponentDefinition(relDirPath string) error {
-	relPath := filepath.Join(relDirPath, string(r.Kind), string(r.Type())+".json")
+func (r RelationshipDefinition) WriteRelationshipDefinition(relDirPath string, fileType string) error {
+	relPath := filepath.Join(relDirPath, fmt.Sprintf("%s-%s.%s", r.Kind, utils.GetRandomAlphabetsOfDigit(3), fileType))
+	if fileType == "yaml" {
```

Comment:
While I understand that you're not considering the file extension here and that this is identified elsewhere, I do wonder if that logic allow for both `.yaml` and `.yml` or if it even depends on the file extension at all.

Reply:
Will update in new PR.